client/app/utils/actions.ts

```diff
@@ -43,7 +43,7 @@ export async function getToken() {
 export async function actionHandler(
   endpoint: string,
   method: "GET" | "POST" | "PUT" | "DELETE" | "PATCH",
-  pathToRevalidate: string,
+  pathToRevalidate?: string,
```

Comment:
I noticed that there was some weird visual judder happening in the page every time we searched/fetched data and tracked it down to the pathToRevalidate param in the actionHandler. This visual judder is kind of subtle though it looks pretty bad IMO.


I made that an optional param as I didn't think it was needed for this feature, though I thought I'd flag this incase someone thinks otherwise. 

https://nextjs.org/docs/app/api-reference/functions/revalidatePath